docs/product/explore/session-replay/replay-details.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Replay Details"
+sidebar_order: 72
+description: "Learn more about how information is organized on the Replay Details page and how to share and delete replays."
+---
+
+<Include name="session-replay-web-report-bug.mdx" />
+
+Every replay has a detailed view that contains the embedded video player and rich debugging context. Playing back the video will allow you to see every user interaction in relation to frontend and backend errors,console messages, DOM events, and network requests and more. It’s like having [DevTools](https://developer.chrome.com/docs/devtools/overview/) active for your production user sessions. Almost every component on this page is connected through timestamps. See the breakdown of each component and why it’s valuable:
+
+![Session Replay details user interface](./img/replay-details.png)
+
+- **Replay Player:** Video-like reproduction of a user session. This is where you can visualize exactly what actions the user took during a user session, and how the application behaved within this specific user’s environment: including device, OS, latency, settings, and so on. Most importantly, you can see which actions lead to an error and take the guesswork out of debugging.
+  - By default, the Session Replay SDK is configured to redact all text, user input and media elements. See all privacy configuration options [here](/platform-redirect/?next=/session-replay/privacy/).
+
+* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place. Breadcrumbs are synced with the replay player and will auto-scroll as the video plays. 
+Specifically:
+- user clicks or taps (including rage and dead clicks)
+  - augmented with surrounding HTML source code (web & react native) to help users understand which event was targeted.
+- user inputs
+- [navigations, page loads, view changes](/product/insights/overview/transaction-summary/#what-is-a-transaction)
```

Comment:
I think we should use a span-focused page, we're trying to move away from using `transaction`

Reply:
update to be: `/product/sentry-basics/distributed-tracing/`